browser-api/src/main/java/com/duckduckgo/browser/api/autocomplete/AutoCompleteSettings.kt

```diff
@@ -14,11 +14,8 @@
  * limitations under the License.
  */
 
-package com.duckduckgo.app.privatesearch
+package com.duckduckgo.browser.api.autocomplete
 
-import com.duckduckgo.navigation.api.GlobalActivityStarter.ActivityParams
-
-/**
- * Use this model to launch the PrivateSearch screen
- */
-object PrivateSearchScreenNoParams : ActivityParams
+interface AutoCompleteSettings {
+    var autoCompleteSuggestionsEnabled: Boolean
```

Comment:
```suggestion
    val autoCompleteSuggestionsEnabled: StateFlow<Boolean>
```
Two changes proposed:
1. This property should be immutable. The setting is only changed from within the `app` module and the setter doesn't need to be exposed outside.
2. Use a flow, why not, the setting could change dynamically.